datalad/metadata/search.py

```diff
@@ -320,7 +325,8 @@ def pretty_str(s):
                            for k, v in s.items()])
     elif isinstance(s, text_type):
         try:
-            return s.encode('utf-8')
+            b = s.encode('utf-8')
+            return (os.linesep + "  ").join(_lines_regex.split(b))
```

Comment:
note to myself... we really need tests for cmdline rendering calls... I am even thinking if somehow we should automate at least smoke testing of them for any call of the Python API interface